client/src/components/Search/Search.js

```diff
@@ -56,31 +53,31 @@ export default class Search extends React.Component {
         }).sort())];
 
         return (
-                <div className="search-results-page">
-                    <div className="map-column">
-                        <MapResults monuments={monuments} zoom={lat && lon ? 10 : 4} center={lat && lon ? [lat, lon] : null}/>
-                    </div>
-                    <div className="search-column">
-                        <div className="search-header">
-                            <Filters onChange={filters => onFilterChange(filters)}
+            <div className="search-results-page">
+                <div className="map-column d-none d-sm-none d-md-flex">
```

Comment:
```suggestion
                <div className="map-column d-none d-md-flex">
```

`d-sm-none` was redundant because `d-none` covers it and `d-md-flex` makes everything md and above flex